source_modelling/srf.py

```diff
@@ -366,18 +366,40 @@ def write_srf_point(srf_file: TextIO, point: pd.Series) -> None:
     point : pd.Series
         The point to write.
     """
+    index = point.name
+    slipt1 = (
```

Comment:
Maybe worth adding a comment to clarify what is happening here?

Reply:
yes, I will also change the code to use indexing that makes more sense.

Reply:
Actually nvm I cannot do that because of how the data is read, I'll write an explanatory comment for the indexing.